### PR TITLE
LPS-160858 Implement update-layout-page-template-entry endpoint to poshi

### PR DESCRIPTION
**Pull request by Luizcarlosqueiroz**
**Ticket**: [LPS-160858](https://issues.liferay.com/browse/LPS-160858)

Motivation
=======
Implement update-layout-page-template-entry endpoint to poshi

Proposed Solution
========
Implement this endpoint to poshi, allowing to set the Display Page Template as default using the JSON API. The usage would  like like this:
```
JSONLayoutpagetemplate.setLayoutPageTemplateEntryAsDefault(
	groupName = "Test Site Name",
	layoutPageTemplateEntryName = "Display Page Name",
	type = "Display Page Template");
```

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by Luizcarlosqueiroz** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/macros/json/layoutpagetemplate/JSONLayoutpagetemplateAPI.macro [hunk 1]
```diff
@@ -186,4 +186,20 @@ definition {
 		return "${masterLayoutPlid}";
 	}
 
+	@summary = "Set the layoutPageTemplateEntry as default by using layoutPageTemplateEntryId"
+	macro _setDisplayPageTemplateEntryAsDefault {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/json/layoutpagetemplate/JSONLayoutpagetemplateAPI.macro:190]: _setLayoutPageTemplateEntryAsDefault, there are not only display page template but also master page template

File: portal-web/test/functional/com/liferay/portalweb/macros/json/layoutpagetemplate/JSONLayoutpagetemplateAPI.macro [hunk 2]
```diff
@@ -186,4 +186,20 @@ definition {
...
+		var portalURL = JSONCompany.getPortalURL();
+
+		var curl = '''
+			${portalURL}/api/jsonws/layout.layoutpagetemplateentry/update-layout-page-template-entry \
+				-u test@liferay.com:test \
+				-d layoutPageTemplateEntryId=${layoutPageTemplateEntryId} \
+				-d defaultTemplate=true
+		''';
+
+		com.liferay.poshi.runner.util.JSONCurlUtil.post("${curl}");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/json/layoutpagetemplate/JSONLayoutpagetemplateAPI.macro:202]: JSONCurlUtil.post("${curl}"); is enough

**Comment by Tim-Cao** [comment]
See changes @Luizcarlosqueiroz
https://github.com/liferay-echo/liferay-portal/pull/9333

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**